src/olympia/constants/categories.py

```diff
@@ -408,6 +408,7 @@ def _immutable(self, *args):
         'general': StaticCategory(name=_(u'General'))
     },
 }
+CATEGORIES_NO_APP[_ADDON_PERSONA] = copy.copy(CATEGORIES_NO_APP[ADDON_STATICTHEME])
```

Comment:
iirc it needs to be:
```
CATEGORIES_NO_APP[_ADDON_PERSONA] = {
    slug: copy.copy(cat)
    for slug, cat in CATEGORIES_NO_APP[ADDON_STATICTHEME].items()}
``` 
as before because the `StaticCategory` needs to be copied too (`copy.copy` will shallow copy) - it gets modified in the next few lines.